README.md

```diff
@@ -1327,24 +1340,55 @@ The resulting file information will be stored in the database in two places:
 1. the `submissions` table
 2. the `user_files` table
 
-The saved JSON is store in the `submissions` table with the field name being the key.
+The `submissions` table is where the file is associated with a submission as part of the
+JSON data. It is stored in the `input_data` JSON and the key name is the input field name that the
+Dropzone instance was associated with.
 
 Example JSON:
 
 ```JSON
 {
   /* other content */
-  "license_upload": "[34,47]"
+  "uploadDocuments": "[\"36ecebb5-c74b-430c-abd7-86b9c211aa71\",\"d2fb9fb6-510a-4cf6-bf2f-04e81daf6a13\",\"c578bf3b-af04-4c8e-9dc0-3afa6946ceb6\"]"
   /* other content */
 }
 ```
 
-This indicates that there were two files uploaded via the `license_upload` widget. Their file ids
-are `34` and `47`, and can be looked up in the `user_files` table (detailed below) using those ids.
+This indicates that there were three files uploaded via the `uploadDocuments` widget. Their file ids
+are `36ecebb5-c74b-430c-abd7-86b9c211aa71`, `d2fb9fb6-510a-4cf6-bf2f-04e81daf6a13`,
+and `c578bf3b-af04-4c8e-9dc0-3afa6946ceb6`. These files can be looked up in the `user_files` table
+using those ids.
+
+The `user_files` table holds information about files a user uploaded as part of their submission.
+The table includes:
+
+* `file_id` - the ID of the file (UUID)
+* `filesize` - size of file, in bytes
+* `submission_id` - ID of the submission the file relates to
+* `created_at` - time file was uploaded
+* `original_name` - original name of the file at the time of upload
+* `doc_type_use` - the document type use label for the file
```

Comment:
Confused about the difference between doc_type_use and doc_type_label?

Reply:
Oh, that's a typo - it should be `doc_type_label` - I'll fix it.  We are hoping that the name `doc_type_label` is unique enough that folks don't think it's technical metadata like mimetype.